eslint.config.js

```diff
@@ -171,13 +169,19 @@ const config = [
       "@typescript-eslint/no-unsafe-call": "off",
     },
   },
-  // Ignore the following `.js` files from linting for now.
+  // Ignore the following files for now.
   {
     files: [
-      "eslint.config.js",
       "src/app/global-error.js",
-      "src/db/**/*.js",
-      "src/scripts/**/*.js",
+      "src/db/migrations/*.js",
+      "src/scripts/build/*.js",
```

Comment:
Would the build scripts need lots of ignore comments? Otherwise, it might be good to catch potential bugs in those as well?

Reply:
Created PR #5622 which extends TSLint coverage to include these files.